commands/Fun/choice.js

```diff
@@ -1,4 +1,4 @@
-exports.run = (client, msg, choices) => {
+exports.run = (client, msg, ... [choices]) => {
```

Comment:
Ack, choices should actually be just one array.

`!choice Red|Black|Orange red` => `['Red', 'Black', 'Orange red']`.